packages/wrangler/src/index.tsx

```diff
@@ -1225,6 +1235,10 @@ export async function main(argv: string[]): Promise<void> {
           default: true,
           describe: "Enable live preview",
           type: "boolean",
+        })
+        .option("minify", {
+          describe: "Utilize ESBuild minification on script",
+          type: "boolean",
```

Comment:
Doesn't preview just use remote Dev? If we don't have parity with `Dev` I think it would make more sense to just have it deprecated and give the user a message for the `Dev` command

Reply:
That's what we currently do, but we don't want to introduce any new flags for this command (we already don't have all the flags from `dev`)